docs/int/faq/errors.mdx

```diff
@@ -8,15 +8,15 @@ description: Errors & Resolutions
 All operators should try to restart their nodes and should check if they are on the latest stable version before attempting anything other configuration change as we are still in beta and frequently releasing fixes. You can restart and update with the following commands:
 
 ```
-docker-compose down
+docker compose down
```

Comment:
`docker-compose` is going to be deprecated in june, 23. Thanks @dB2510 for pointing this out

https://www.docker.com/blog/new-docker-compose-v2-and-v1-deprecation/